### DISCUSSION TITLE
full stopp after ladning

### DISCUSSION THREAD

**Discussion by LukasGNN** [flybywiresim/aircraft#3964] [Q&A] [answered]
Hi,

i have experienced that the aircraft full stops after landing. There is no manuall brake applied og revers thrust,  anyway is stops.
Another thing is that spoilers remain deployed, they do not disangage automatically.

Next case:
Yesterday i had problems with keeping speed during taxing. The aircraft stops after a few meters..i need to apply approx 40% of thrust to keep the aircraft rolling...
any idea whats going on??

**Comment by MapleNAV-TCASFAULT** [comment]
First, spoilers need to be manually retracted. Second, if thrust is constantly needed, it may be caused by the wet runway conditions made by Asobo.

**Comment by mojoair** [comment, marked as answer]
If you are using autobrake for landing, it will bring aircraft to full stop unless you disconnect prior to full stop. Tapping the manual brakes will disengage the autobrake and you'd manually brake from there (usually once it's slowed down to your desired runway exit speed).

As other user posted, retract spoilers/flaps manually during cleanup phase once you exit runway. It's not an automated process. Taxi thrust needed is based on weather/ground conditions. On a dry normal day, you should only need thrust to get moving and it should taxi on Idle thrust.

**Comment by LukasGNN** [comment]
all this makes sens. It was wet conditions that day. :)
Thanks a lot for reply !
